plugins/modules/ec2_instance.py

```diff
@@ -2146,6 +2278,8 @@ def main():
             ["image_id", "image"],
             ["exact_count", "count"],
             ["exact_count", "instance_ids"],
+            ["tenancy", "placement"],
+            ["placement_group", "placement"],
```

Comment:
@hakbailey Added the checks to validate only one of tenancy/placement_group or placement is defined